showcase/src/elm/Utils/FileParser.elm

```diff
@@ -0,0 +1,389 @@
+module Utils.FileParser exposing (elliefy)
+
+
+{- | File-related Logic
+-}
+
+import Parser exposing ((|.), (|=), DeadEnd, Parser, Problem(..), Step(..))
+import Url.Builder
+
+
+
+-------------------
+-- PARSING STUFF
+-- Elliefy makes the code renderable in Ellie. WIP
```

Comment:
```suggestion
-- Elliefy makes the code renderable in Ellie.
```

It's easy to forget to remove things such as `WIP` or `TODO` (I do this a lot) - it's better to create github issues to track these things.

In any case, we also know it's `WIP` because it hasn't been merged :)